.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,189 @@
+name: Create Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      source_ref:
+        description: 'Source branch, commit, or tag to create the release branch from'
+        default: 'develop'
+        type: string
+      release_version:
+        description: 'The release_version used for the release branch name, e.g. release/vx.x.x'
+        default: 'vx.x.x'
+        required: true
+        type: string
+      pre_release_version:
+        description: "Pre-Release version, e.g. 'b1', will be added behind the release_version in setup.py."
+        required: false
+        type: string
+      release_branch_exists:
```

Comment:
If the step on line 66 created the release branch (if it doesn't exist) _prior_ to the checkout steps, would this still be necessary as an input?